src/Metadata/HttpOperation.php

```diff
@@ -112,7 +113,6 @@ public function __construct(
         ?bool $paginationClientPartial = null,
         ?bool $paginationFetchJoinCollection = null,
         ?bool $paginationUseOutputWalkers = null,
-        ?array $paginationViaCursor = null,
```

Comment:
Why changing the position of this parameter?

Reply:
Because this parameter was in the "abstract operation arguments" group of parameters.
It doesn't make sense to keep it here, and can cause maintenance issue later.